apps/server/src/modules/school/uc/school.uc.spec.ts

```diff
@@ -63,6 +63,16 @@ describe('SchoolUc', () => {
 			authService.checkPermissionByReferences.mockImplementation(() => Promise.resolve());
 		});
 
+		beforeEach(() => {
+			schoolService.getSchoolById.mockResolvedValue(
+				new SchoolDO({
```

Comment:
For DO test it should be exists factories. Otherwise it will be harder and harder to refactor test after chances.